src/index.html

```diff
@@ -30,6 +30,7 @@
":"cert-manager","version":"0.9.0"},{"name":"cert-operator","version":"0.1.0"},{"name":"chart-operator","version":"0.7.0"},{"name":"cluster-autoscaler","version":"1.16.2"},{"name":"cluster-operator","version":"2.0.2"},{"name":"containerlinux","version":"2247.6.0"},{"name":"coredns","version":"1.6.5"},{"name":"docker","version":"18.06.1"},{"name":"etcd","version":"3.3.17"},{"name":"external-dns","version":"0.5.11"},{"name":"kiam","version":"3.4.0"},{"name":"kube-state-metrics","version":"1.9.2"},{"name":"kubernetes","version":"1.16.3"},{"name":"metrics-server","version":"0.3.3"},{"name":"node-exporter","version":"0.18.1"},{"name":"vault","version":"0.10.3"}],"timestamp":"2020-01-29T12:00:00Z","version":"11.0.0"},{"active":true,"changelog":[{"component":"aws-operator","description":"Fix AWS resource tags."},{"component":"aws-operator","description":"Allow network traffic between Node Pools."},{"component":"cluster-operator","description":"Moved default app list from cluster-operator code to release repository."},{"component":"cert-exporter","description":"Remove CPU limits."},{"component":"chart-operator","description":"Handle timeouts pulling chart tarballs."},{"component":"kube-state-metrics","description":"Upgraded to kube-state-metrics new release 1.9.0"},{"component":"net-exporter","description":"Changed Priority Class to system-node-critical for net-exporter deployed in TC."},{"component":"node-exporter","description":"Change Priority Class to system-node-critical"},{"component":"metrics-server","description":"Updated manifests for Kubernetes 1.16."},{"component":"cert-manager","description":"Updated helm chart to use a same image registry for parent and cluster-issuer subchart."},{"component":"cluster-autoscaler","description":"Increase memory request and limits to 400MB."},{"component":"coredns","description":"Fix HPA manifests for Kubernetes 1.16."},{"component":"external-dns","description":"Updated external-dns app version to v0.5.18."},{"component":"kiam","description":"Updated kiam app version to v3.5."}],"components":[{"name":"app-operator","version":"1.0.0"},{"name":"aws-operator","version":"8.1.1"},{"name":"calico","version":"3.10.1"},{"name":"cert-operator","version":"0.1.0"},{"name":"cluster-operator","version":"2.1.1"},{"name":"containerlinux","version":"2247.6.0"},{"name":"docker","version":"18.06.1"},{"name":"etcd","version":"3.3.17"},{"name":"kubernetes","version":"1.16.3"},{"name":"vault","version":"0.10.3"},{"name":"cert-exporter","version":"1.2.1"},{"name":"cert-manager","version":"0.9.0"},{"name":"chart-operator","version":"0.11.4"},{"name":"cluster-autoscaler","version":"1.16.2"},{"name":"coredns","version":"1.6.5"},{"name":"external-dns","version":"0.5.18"},{"name":"kiam","version":"3.5.0"},{"name":"kube-state-metrics","version":"1.9.0"},{"name":"metrics-server","version":"0.3.3"},{"name":"net-exporter","version":"1.5.1"},{"name":"node-exporter","version":"0.18.1"}],"timestamp":"2020-02-05T12:00:00Z","version":"11.0.1"}],
```

Comment:
Can we make that easier to read and maintain? At some point we will have to remove a release and add a new one, I'm sure.

Reply:
hmm.. I was thinking what we should do generally is just copy the full release endpoint output from a valid installation, and not manage it too much manually.